### PR TITLE
Update colors of Tetris blocks

### LINKED ISSUE #3: Change colors of Tetris blocks
Based on some conversations that occurred during the [last sprint meeting](https://github.com/githubschool/hot-garbage/issues/7), the color palette of the Tetris blocks needs to be modified. 

The following colors should be used for the shapes: 

- D72638
- 3F88C5
- F49D37
- 140F2D
- F22B29

Examples of the colors can be found below:

image

### PR DESCRIPTION
**Pull request by dumpsterfire**
This PR resolves #3 by updating the colors used for the Tetris blocks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,11 +113,11 @@
...
-    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'pink'   };
-    var l = { size: 3, blocks: [0x4460, 0x0E80, 0xC440, 0x2E00], color: 'salmon' };
-    var o = { size: 2, blocks: [0xCC00, 0xCC00, 0xCC00, 0xCC00], color: 'lime' };
-    var s = { size: 3, blocks: [0x06C0, 0x8C40, 0x6C00, 0x4620], color: 'teal'  };
+    var i = { size: 4, blocks: [0x0F00, 0x2222, 0x00F0, 0x4444], color: '#D72568'   };
+    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: '#3F88C5'   };
+    var l = { size: 3, blocks: [0x4460, 0x0E80, 0xC440, 0x2E00], color: '#F49D37' };
+    var o = { size: 2, blocks: [0xCC00, 0xCC00, 0xCC00, 0xCC00], color: '#140F2D' };
+    var s = { size: 3, blocks: [0x06C0, 0x8C40, 0x6C00, 0x4620], color: 'F22B29'  };
     var t = { size: 3, blocks: [0x0E40, 0x4C40, 0x4E00, 0x4640], color: 'coral' };
```
Comment (beardofedu) [inline index.html]: @rwnfoo How should we address the colors here? We identified the colors to use in #3 but we we potentially need 2 more colors. Do we have additional colors we want to use or should we reuse colors for the blocks?

Comment (beardofedu) [inline index.html]:
```suggestion
    var t = { size: 3, blocks: [0x0E40, 0x4C40, 0x4E00, 0x4640], color: '#D72568' };
```

Assuming @rwnfoo is ok with duplicate colors for Tetris blocks, @dumpsterfire please merge this and the other suggested change in.

File: index.html [hunk 2]
```diff
@@ -113,11 +113,11 @@
...
     var z = { size: 3, blocks: [0x0C60, 0x4C80, 0xC600, 0x2640], color: 'olive'    };
```
Comment (beardofedu) [inline index.html]:
```suggestion
    var z = { size: 3, blocks: [0x0C60, 0x4C80, 0xC600, 0x2640], color: '#3F88C5'    };
```

**Review by beardofedu — CHANGES_REQUESTED** [review]
Think this is pretty close to being able to be shipped. Once we address the color 🎨 issue we can merge this in. 🎉

**Comment by rwnfoo** [comment]
I am ok with duplicating colors for the two remaining blocks 

@beardofedu @dumpsterfire

File: index.html (hunk 1 above)
Comment (whatupfoo) [reply index.html]: I am ok with duplicating colors for the two remaining blocks
@beardofedu @dumpsterfire

**Review by beardofedu — APPROVED** [review]
This is probably your best work ever. You deserve a raise! 💰

**Pull request merged**